resources/views/components/layout.blade.php

```diff
@@ -57,38 +57,33 @@
                         </div>
 
                         {{ $slot }}
-                  
+
                 </div>
         </div>
 </div>
     <script>
 
         window.addEventListener('load',()=>{
-            if(document.getElementById('title').innerHTML=='I tuoi Rider'){
+            if(document.getElementById('title')?.innerHTML=='I tuoi Rider'){
```

Comment:
This fixes a js error when there is no element with `id="title"` on the page. If the element is not present, it will not crash anymore